src/styles/Document.js

```diff
@@ -54,7 +54,7 @@ export default css`
 .renderer {
   padding: 8px 0;
   display: flex;
-  max-height: 500px;
+  max-height: var(--api-resource-example-document-max-height, 500px);
```

Comment:
I believe I would actually change this to:
```
max-height: var(--api-resource-example-document-max-height, initial);
```

So the default behavior doesn't change. Or `unset` could also work